src/app/modules/gb-cohort-selection-module/gb-cohort-selection.module.ts

```diff
@@ -46,7 +46,8 @@ import {GbPedigreeConstraintComponent} from './constraint-components/gb-pedigree
     DropdownModule,
     MessagesModule,
     MultiSelectModule,
-    OverlayPanelModule
+    OverlayPanelModule,
+    InputSwitchModule
```

Comment:
showing a label next to an input switch would be too much space? especially when the query is big and nested. How about just use a basic toggle button? as in here [https://www.primefaces.org/primeng/#/togglebutton](url)

Reply:
There will be some styling change added for [TMT-727](https://jira.thehyve.nl/browse/TMT-727), when we introduce the context aware user friendly box messages. With the description in the top part of the box it will be possible to move the input switch up (so it will be aligned with the text). I guess then it will not be a problem that the switch with the label is taking more space. 
Probably better to discuss the styling changes with Elisa.

Reply:
functionally toggle button and switch input are the same, just the latter occupies less space, always good to save some (precious) space

Reply:
it is addressed here: [TMT-766](https://jira.thehyve.nl/browse/TMT-766)